DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1

```diff
@@ -96,8 +96,8 @@ function Set-TargetResource
         # Port number of the DSC Pull Server IIS Endpoint
         [Uint32]$Port = $( if ($IsComplianceServer) { 7070 } else { 8080 } ),
 
-        # Physical path for the IIS Endpoint on the machine (usually under inetpub/wwwroot)                            
-        [string]$PhysicalPath = "$env:SystemDrive\inetpub\wwwroot\$EndpointName",
```

Comment:
I see that the installation does go to C:\inetpub\wwwroot\psdscpullserver currently. Why is it being changed?